### DISCUSSION TITLE
Firebase 콘솔을 열어서 Authentication 페이지의 Users 탭에서 users가 안보임.

### DISCUSSION THREAD

**Discussion by s570908** [velopert/dealing-with-react-native#14] [Q&A] [answered]
p.448에서 "객체 출력이 잘 이루어졌다면 Firebase 콘솔을 열어서 Authentication 페이지의 Users 탭을 열어보세요. 방금 등록한 정보가 이 페이지에 나타날 것입니다." 라고 하셨는데 실제로 해보니 Users가 보이지 않습니다. signUp user에 대한 객체 출력은 console에 잘 보입니다.  동일한 user email password로 다시 signup을 하면 이미 signUp한 user라고 에러메시지를 내는 것을 봐서는 firebase에 user가 잘 기록되어 있는 것으로 보입니다. 

firebase 콘솔을 열어서 Authentication 페이지의 Users 탭으로 signUp된 user를 보려면 firebase에서 추가 설정을 해야 하는 것인지, 아니면 firebase API의 사용 시에 추가 작업을 해야 하는 것인지 궁금합니다. 

firebase 콘솔을 열어서 Authentication 페이지의 Users 탭으로 signUp된 user를 볼 수 있는 방법을 알려 주시면 감사하겠습니다.

**Comment by s570908** [comment, marked as answer]
프로젝트, 안드로이드 스튜디오, 디바이스 에뮬레이터를 cleanup하고 다시 빌드하니 잘 됩니다.  도서에서 알려 주신대로 차분히 다시 하니까 잘됩니다.

**Comment by s570908** [comment]
도서에서 알려 주신 것이외에 세팅을 한 것이 있습니다. 디바이스 에뮬레이터의 time zone을 서울로 변경하였습니다. 이것 때문에 잘 된 것일 수도 있지 않을까요?

**Reply by velopert** [reply]
타임존이랑은 관계없을 것 같긴한데..
뭔가 구글측에 어떠한 동기화 오류가 있지 않았을까 싶네요~
